AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1383,51 +1392,28 @@ - (void)recursivelyDisplayImmediately
   [self displayImmediately];
 }
 
-//Calling this with the lock held can lead to deadlocks. Always call *unlocked*
-- (void)__setNeedsLayout
+- (void)invalidateCalculatedLayout
```

Comment:
Do we still need this method or can we just fold it into __setNeedsLayout?  Or are we keeping invalidateCalculatedLayout as an external API?